packages/web-components/src/components/ic-pagination-item/ic-pagination-item.tsx

```diff
@@ -0,0 +1,137 @@
+import {
+  Component,
+  Prop,
+  Element,
+  h,
+  Event,
+  EventEmitter,
+} from "@stencil/core";
+import { IcPaginationItemType } from "./ic-pagination-item.types";
+import { IcThemeForeground } from "../../interface";
+
+@Component({
+  tag: "ic-pagination-item",
+  styleUrl: "ic-pagination-item.css",
+  shadow: true,
+})
+export class PaginationItem {
+  @Element() el: HTMLIcPaginationItemElement;
+
+  /**
+   * The current page number.
+   */
+  @Prop() page: number | null;
+  /**
+   * If 'true', the pagination item will be selected.
```

Comment:
Same about the backticks for 'true' in the descriptions in this file :)
```suggestion
   * If `true`, the pagination item will be selected.
```